contracts/tasks/helpers/ContractStorage.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+import low from "lowdb";
+import FileSync from "lowdb/adapters/FileSync";
+
+import type { EContracts, IRegisterContract, IStorageInstanceEntry, IStorageNamedEntry } from "./types";
+
+type TStorage = Record<
+  string,
+  Partial<{ named: Record<string, IStorageNamedEntry>; instance: Record<string, IStorageInstanceEntry> }>
+>;
+
+export class ContractStorage {
```

Comment:
should we add typedoc comments to this and other code inside helpers?

Reply:
Not in this PR but I have a plan to describe all the docs (this is a separate subtask).